src/components/SendModal/ProvisionIdentity/ProvisionIdentityForm/ProvisionIdentityForm.js

```diff
@@ -125,7 +134,8 @@ const ProvisionIdentityForm = (props) => {
     try {
       fromBase58Check(identity);
     } catch (e) {
-      if (!identity.endsWith('@')) {
+      const formattedId = state.parentname ? `${identity}${state.parentname}` : identity;
```

Comment:
If the user submits an I-addr (which should be accepted in cases where the ID being provisioned already exists), the method of adding the parent fqn to the end of the data entered in the ID field will turn the i-address into an ID with fully qualified name of <i-address>.<parent>@. To solve this, I would recommend changing the form to throw an error if an I address is entered and the ID is going to be created with a parent, and I would change the header of the text box to no longer suggest i-addrs are allowed if a parent is going to be appended.